packages/react-native-unimodules/README.md

```diff
@@ -1,3 +1,23 @@
 # react-native-unimodules
 
-Learn more about this library and learn how to install it on the [Installing react-native-unimodules documentation page](https://docs.expo.io/bare/installing-unimodules/).
\ No newline at end of file
+Learn more about this library and learn how to install it on the [Installing react-native-unimodules documentation page](https://docs.expo.io/bare/installing-unimodules/).
+
+## Config with package.json
+
+You can configure the iOS and Android linking using the `unimodules` object in the `package.json`:
+
+```json
+{
+  "unimodules": {
```

Comment:
I wouldn't use `unimodules` anymore, especially in new places. We may want to deprecate `react-native-unimodules` and move this to the core or the adapter and call them "Expo modules".
Maybe we could already use `expo-modules` key? What do you think @ide?

Reply:
When we rename unimodules to Expo modules, we can rename the package.json key too.

Reply:
Agree with both. I'll add that this is a pretty niche feature that'll mostly be used to support monorepo managed workflow projects.